src/redist/targets/GenerateMSIs.targets

```diff
@@ -202,21 +244,48 @@
                       '$(SdkMSICabFile)'" />
   </Target>
 
-  <Target Name="GenerateTemplatesNupkg"
-          DependsOnTargets="GenerateLayout;MsiTargetsSetupInputOutputs;GenerateTemplatesMsi;SignTemplatesMsi"
-          Condition=" '$(OS)' == 'Windows_NT' "
-          Inputs="$(TemplatesMSIInstallerFile);
+  <Target Name="GenerateTemplatesNupkgs"
+          DependsOnTargets="GenerateLayout;MsiTargetsSetupInputOutputs;GenerateTemplatesMsis;SignTemplatesMsis;SetupTemplatesNupkgs"
+          Condition="$(ProductMonikerRid.StartsWith('win')) And !$(Architecture.StartsWith('arm'))"
+          Inputs="@(TemplatesNupkgComponent->'%(MSIInstallerFile)');
                     $(TemplatesInstallerNuspecFile);
                     $(GenerateNupkgPowershellScript)"
-          Outputs="$(TemplatesInstallerNupkgFile)">
+          Outputs="@(TemplatesNupkgComponent->'%(NupkgFile)')">
 
     <Exec Command="powershell -NoProfile -NoLogo $(GenerateNupkgPowershellScript) ^
-                      '$(TemplatesMSIInstallerFile)' ^
-                      '$(FullNugetVersion)' ^
+                      '$(ArtifactsDir)' ^
+                      '%(TemplatesNupkgComponent.MSIInstallerFile)' ^
```

Comment:
Same here.

Reply:
The syntax looks equivalent:
%(TemplatesNupkgComponent.MSIInstallerFile) evaluates to: $(Templates30MSIInstallerFile)